src/js/components/index.d.ts

```diff
@@ -0,0 +1,73 @@
+export * from './Accordion';
```

Comment:
How does this relate to `src/js/index.d.ts`, which seems to be doing a similar thing?

Reply:
In my opinion, wherever we have an `index.js` file that exports members, we should have an `index.d.ts` file in the same directory that exports the types of those members.

At the moment, there is a `src/js/index.js` file and a `src/js/components/index.js` file, and both files export components. It's thus possible that consumers of Grommet can import components from `grommet` or `grommet/components`, and we should provide type bindings for both.

This change adds typings for the pre-existing `src/js/components/index.js`, which these tests rely on.